mteb/tasks/Retrieval/eng/ClimateFEVERRetrieval.py

```diff
@@ -72,3 +72,39 @@ class ClimateFEVERHardNegatives(AbsTaskRetrieval):
       primaryClass={cs.CL}
 }""",
     )
+
+
+class ClimateFEVERv2(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="ClimateFEVER.v2",
+        description="CLIMATE-FEVER is a dataset adopting the FEVER methodology that consists of 1,535 real-world claims regarding climate-change. ",
+        reference="https://www.sustainablefinance.uzh.ch/en/research/climate-fever.html",
+        dataset={
+            "path": "Mina76/climate-fever",
+            "revision": "e438c9586767800aeb10dbe8a245c41dbea4e5f4",
+        },
+        type="Retrieval",
+        category="s2p",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=("2020-12-11", "2020-12-11"),
+        domains=["Academic"],
+        task_subtypes=["Question answering"],
+        license="cc-by-sa-4.0",
+        annotations_creators="human-annotated",
```

Comment:
if the metadata is not filled out in the old one - could you move this up there as well?